### PR TITLE
Add support for doorbells

### PR DESCRIPTION
**Pull request by NielsLeenheer**
Add the doorbell service for doorbells defined in Domoticz. 

It is not possible to use the doorbell service on it's own as you need to combine it with a different service. The most logical service I could come up with was the stateless programmable switch service as the doorbell basically also is a switch. 

When combined in this way the doorbell shows up in the Home app and allows you to turn on notifications. So whenever somebody pressed the doorbell you get a notification that says "{location} doorbell rang", where the location is name of the room where the doorbell is added.

Also, when in the same room there is also a camera present, it will basically act as a video doorbell. It shows a snapshot in the notification and allows you to look at the live video stream.

Combining it with a stateless programmable switch also offers an added bonus, but you can now configure the switch to activate a scene or turn on and off individual devices.

TLDR; doorbell with notifications, video from cam in the same room and programmable switch.

### CODE REVIEW

**Comment by NielsLeenheer** [comment]
BTW, I've not bothered with getting it working without MQTT, as it will not properly work anyway.
